### DISCUSSION TITLE
Transfer a ticket from one event to another

### DISCUSSION THREAD

**Discussion by raphaeldeux** [pretix/pretix#3998] [Q&A]
Hello,

I am wondering, is it possible to transfer a ticket from one event to another ?

**My usecase :**
John is organizing 2 workshops (on 24/12 and 12/01).
Paul has a ticket for the workshop on 24/12 but can not come, but is now free for 12/01.
John would like to transfer the ticket of Paul from the workshop 24/12 to the workshop 12/01.

Is it possible to do so ?

**Comment by raphaelm** [comment]
it's possible only if the two workshops are part of the same event series

**Reply by raphaeldeux** [reply]
Ok thank you for the answer ! 

My events are unique, but I should have made them a serie...
Do you know if it's possible to transfer unique events (with their participants) into one serie ?
